src/css/aclmanagement.css

```diff
@@ -104,6 +104,12 @@ html, body {
     margin: 4px 0 0 3px;
 }
 
+#graphRoleCell, #pluginRoleCell, #systemRoleCell, #statementRoleCell {
+    overflow-wrap: anywhere;
```

Comment:
Not sure why do you need this? You should be cautious with these properties because many of them are inheritted by the children and the things can get messy

Reply:
My idea here was to override the `break-word` set for all the `td.data`, because I noticed that long role names with underscores (no hyphens or spaces) don't break and overflow into the next cell. But I don't know how realistic it is to expect that users will use only underscores.